index.html

```diff
@@ -5,8 +5,7 @@
   <meta name="viewport" content="width=device-width">
   <title>OpenHistoricalMap</title>
   <script src="./index.js" type="text/javascript"></script>
-  <link rel="stylesheet" href="./index.css">
-  <style type="text/css">
+  <style>
```

Comment:
esbuild generates https://embed.openhistoricalmap.org/index.css by concatenating any CSS file required by JavaScript (plus any CSS files we manually specify). Currently, all that we require is MapLibre’s stylesheet, so that’s all it contains:

https://github.com/OpenHistoricalMap/openhistoricalmap-embed/blob/63c575b5d95465d53d05425bce6071723099f30a/index.js#L3

We could split the inline stylesheet into an index.css (so that it resolves to something locally) and add it to the esbuild command (so that it gets concatenated with MapLibre’s stylesheet).